sdks/python/apache_beam/transforms/util.py

```diff
@@ -196,6 +199,16 @@ def _empty(actual):
   return _empty
 
 
+def contains_in_any_order(expected):
```

Comment:
Good point. I think this is redundant. I will do more cleanups of the examples, I will remove this function together with those changes.